src/test/java/api/PrecedingSucceedingTitlesApiExamples.java

```diff
@@ -132,6 +141,135 @@ public void canUpdateAnInstanceWithUnconnectedSucceedingTitles()
       null);
   }
 
+  @Test
+  public void titleIsRequiredToCreateAnInstanceWithUnconnectedSucceedingTitles()
+    throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+
+    final JsonObject succeedingTitle = createSemanticWebUnconnectedTitle(UUID.randomUUID().toString());
+    succeedingTitle.remove("title");
+
+    final JsonObject smallAngryPlanetJson = smallAngryPlanet(UUID.randomUUID())
+      .put(SUCCEEDING_TITLES_KEY, new JsonArray()
+        .add(createSemanticWebUnconnectedTitle(UUID.randomUUID().toString()))
+        .add(succeedingTitle));
```

Comment:
Why are two unconnected titles needed to demonstrate this behaviour?

Reply:
One of them is a valid unconnected relationship but another one is not valid.

Reply:
Why does this test need a valid relationship as well as an invalid one to demonstrate the behaviour?

Reply:
To verify that validation checks each item in array, not only the first one.

Reply:
Ok, please could a comment be added to this test to explain that reasoning